descriptors/ModuleDescriptor-template.json

```diff
@@ -1438,6 +1438,18 @@
     {
       "id": "contributor-name-types",
       "version": "1.3"
+    },
+    {
+      "id": "source-manager-job-executions",
+      "version": "3.4"
+    },
+    {
+      "id": "organizations.organizations",
+      "version": "1.2"
+    },
+    {
+      "id": "data-import-converter-storage",
+      "version": "1.5"
```

Comment:
For this log:
sidecar-mod-orders,/circulation/requests/285d0a68-00f8-45e6-ba50-46da3fd007a3/move,1

we should add this interface
https://github.com/folio-org/mod-circulation/blob/master/descriptors/ModuleDescriptor-template.json#L81

Reply:
and we can add it below circulation interface

Reply:
thanks to notice, added it